### PR TITLE
Web 1838 rm6119 furniture and associated services

### PR DESCRIPTION
**Pull request by dorota-hawro**

### CODE REVIEW

File: graph/RM6119_Furniture_and_Associated_Services.cypher [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+
+
+// ANSWERS
+
+// Answers: level 1
+(ansProduct:Answer {
+    uuid: '518afdfd-c077-41f5-842c-c42b35ad4c3b',
+    text: 'Product',
+    hint: 'A product is an item you can buy. It\'s a one-off purchase and involves no ongoing contract.'
```
Comment (Chee-Ng) [inline graph/RM6119_Furniture_and_Associated_Services.cypher]: remove all the full stop at the end of the sentence for the hint text

File: graph/RM6119_Furniture_and_Associated_Services.cypher [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+    type: 'LIST'
+}),
+
+(qstnWhatService:Question {
+    uuid: '06a14aa8-7b90-4e2a-a249-a55e8458e644',
+    text: 'What service do you need?',
+    type: 'LIST'
+}),
+
+// (qstnRepairAndRenovation:Question {
```
Comment (Chee-Ng) [inline graph/RM6119_Furniture_and_Associated_Services.cypher]: removed unused code

**Review by Chee-Ng — CHANGES_REQUESTED** [review]

File: graph/RM6119_Furniture_and_Associated_Services.cypher (hunk 2 above)
Comment (dorota-hawro) [reply graph/RM6119_Furniture_and_Associated_Services.cypher]: done

File: graph/RM6119_Furniture_and_Associated_Services.cypher (hunk 1 above)
Comment (dorota-hawro) [reply graph/RM6119_Furniture_and_Associated_Services.cypher]: done

**Review by Chee-Ng — APPROVED** [review]

**Pull request merged**